client/my-sites/checkout/src/components/checkout-main-content.tsx

```diff
@@ -980,13 +1008,22 @@ const StepContainerV2CheckoutFixer = styled.div< { isLargeViewport: boolean } >`
 				}
 			}
 
-			.checkout-summary-area {
+			.checkout__summary-area,
+			.checkout__summary-body {
 				max-width: 100%;
 			}
 
+			.checkout__summary-area {
+				min-width: 300px;
+			}
+
 			.checkout__summary-body {
 				margin: 0;
 			}
+
+			.checkout__summary-features {
```

Comment:
Can we add back a bit of space between the summary and the products?

| Prod | This PR |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/2dd63eb3-6e41-4f0f-9919-0def9accce46) | ![image](https://github.com/user-attachments/assets/b7ca968e-bbab-4b10-a10e-e05067aa688d) |

It's currently 24px on prod:

![image](https://github.com/user-attachments/assets/01df1174-c93b-4d5e-9f40-ec617c33b732)

Reply:
[Added it here](https://github.com/Automattic/wp-calypso/pull/102513/commits/5b141ff2e02e0be0cb5e72d71094929b2b6c7123):

![image](https://github.com/user-attachments/assets/1e026bec-5282-4e83-a8b1-d8698541dc28)